test/videojs-playlist-ui.test.js

```diff
@@ -372,8 +372,12 @@ test('tracks when an ad is playing', function() {
 // -----------
 
 test('changes the selection when tapped', function(test) {
+  var playCalled = false;
   player.playlist(playlist);
-  player.playlistUi();
+  player.playlistUi({'playOnClick': true});
```

Comment:
I think we should keep a test where this new setting is false so we verify both variations work properly.